pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -9235,11 +9426,12 @@ def test_multifile_read_errors(read_func, filelist):
 def test_multifile_read_check(hera_uvh5, tmp_path):
     """Test setting skip_bad_files=True when reading in files"""
 
-    uvTrue = hera_uvh5
+    uvTrue = hera_uvh5.copy()
+
     uvh5_file = os.path.join(DATA_PATH, "zen.2458661.23480.HH.uvh5")
 
     # Create a test file and remove header info to 'corrupt' it
-    testfile = str(tmp_path / "zen.2458661.23480.HH.uvh5")
+    testfile = os.path.join(tmp_path, "zen.2458661.23480.HH.uvh5")
```

Comment:
thanks for fixing this! I found and fixed a bunch of these at one point, but obviously missed some.